telegram/ext/filters.py

```diff
@@ -192,12 +195,118 @@ def filter(self, message):
     class _Document(BaseFilter):
         name = 'Filters.document'
 
+        class Category(BaseFilter):
```

Comment:
Please use the convention of lowercasing filters.

Reply:
I thought I use the PEP 8 Style here, because it's a class to call. It's different to the other filters yet